tests/apt/test_protocol.py

```diff
@@ -440,15 +441,45 @@ def test_AptMessage_MGMSG_MOT_MOVE_COMPLETED_from_bytes() -> None:
     assert msg.chan_ident == 0x01
 
 
-def test_AptMessage_MGMSG_MOT_MOVE_COMPLETED_to_bytes() -> None:
-    msg = AptMessage_MGMSG_MOT_MOVE_COMPLETED(
+def test_AptMessage_MGMSG_MOT_MOVE_COMPLETED_6_BYTES_to_bytes() -> None:
+    msg = AptMessage_MGMSG_MOT_MOVE_COMPLETED_6_BYTES(
         destination=Address.HOST_CONTROLLER,
         source=Address.BAY_1,
         chan_ident=ChanIdent.CHANNEL_1,
     )
     assert msg.to_bytes() == bytes.fromhex("6404 0100 01 22")
 
 
+def test_AptMessage_MGMSG_MOT_MOVE_COMPLETED_20_BYTES_from_bytes() -> None:
+    msg = AptMessage_MGMSG_MOT_MOVE_COMPLETED_20_BYTES.from_bytes(
+        # Example message from testing with a real K10CR1 waveplate. Not seen on official documentation.
+        bytes.fromhex("6404 0e00 81 50 0100 68aaa001 0000 0000 30000080")
+    )
+    assert msg.message_id == 0x0464
+    assert msg.chan_ident == 0x01
+    assert msg.destination == 0x01
+    assert msg.source == 0x50
+    assert msg.position == 0x01A0AA68
+    assert msg.velocity == 0x00
+    assert msg.motor_current == 0x00
+    assert msg.status == UStatus(INMOTIONCCW=True, INMOTIONCW=True, ENABLED=True)
+
+
+def test_AptMessage_MGMSG_MOT_MOVE_COMPLETED_20_BYTES_to_bytes() -> None:
+    msg = AptMessage_MGMSG_MOT_MOVE_COMPLETED_20_BYTES(
+        chan_ident=ChanIdent.CHANNEL_1,
+        destination=Address.HOST_CONTROLLER,
+        source=Address.GENERIC_USB,
+        position=0x01A0AA68,
+        velocity=0x00,
+        motor_current=pnpq_ureg.milliamp * 0,
```

Comment:
The style you'll see in [the pint manual](https://pint.readthedocs.io/en/stable/getting/overview.html) writes this in the same order we would use in English or Japanese:

```suggestion
        motor_current=0 * pnpq_ureg.milliamp,
```